src/commands/Announcement/announcement.ts

```diff
@@ -27,32 +27,32 @@ export default class extends SkyraCommand {
 	private readonly messages: WeakMap<GuildMessage, GuildMessage> = new WeakMap();
 
 	public async run(message: GuildMessage, [announcement]: [string]) {
-		const [channelID, embedEnabled, language] = await message.guild.readSettings((settings) => [
+		const [channelID, embedEnabled, t] = await message.guild.readSettings((settings) => [
 			settings[GuildSettings.Channels.Announcements],
 			settings[GuildSettings.Messages.AnnouncementEmbed],
 			settings.getLanguage()
 		]);
-		if (!channelID) throw language.get(LanguageKeys.Commands.Announcement.SubscribeNoChannel);
+		if (!channelID) throw t(LanguageKeys.Commands.Announcement.SubscribeNoChannel);
 
 		const channel = message.guild.channels.cache.get(channelID) as TextChannel;
-		if (!channel) throw language.get(LanguageKeys.Commands.Announcement.SubscribeNoChannel);
+		if (!channel) throw t(LanguageKeys.Commands.Announcement.SubscribeNoChannel);
 
-		if (!channel.postable) throw language.get(LanguageKeys.System.ChannelNotPostable);
+		if (!channel.postable) throw t(LanguageKeys.System.ChannelNotPostable);
 
 		const role = await announcementCheck(message);
-		const header = language.get(LanguageKeys.Commands.Announcement.Announcement, { role: role.toString() });
+		const header = t(LanguageKeys.Commands.Announcement.Announcement, { role: role.toString() });
```

Comment:
Creative naming for the header...please rename the key to Header or AnnouncementHeader or something more...descriptive..